SecurityManager/Clients.Web/SecurityManager.Clients.Web.csproj

```diff
@@ -11,9 +11,6 @@
     <AssemblyName>Remotion.SecurityManager.Clients.Web</AssemblyName>
     <RootNamespace>$(AssemblyName)</RootNamespace>
   </PropertyGroup>
-  <ItemGroup>
-    <None Update="..\SecurityManager AGPL.licenseheader" Pack="true" PackagePath="res/" Visible="false" />
-  </ItemGroup>
```

Comment:
still need this, just switch to remoton lgpl file.

Reply:
It delivers the lhm file to the res folder on the consumer side, so they, too have the correct header when our res files are linked in their solution.